zebra/interface.c

```diff
@@ -92,6 +90,24 @@ static void if_zebra_speed_update(struct thread *thread)
 	}
 
 	if (changed || new_speed == UINT32_MAX) {
+#define SPEED_UPDATE_COUNT_MAX 48
+		/*
```

Comment:
I like the symbolic constant here, but it would be better if the actual update timer ival (5s) was defined symbolically as well and the COUNT_MAX then defined like `#define SPEED_UPDATE_COUNT_MAX (4 * 60/SPEE..._SECS)`

Reply:
fixed